Functions/xASL_quant_Basil.m

```diff
@@ -0,0 +1,231 @@
+function [CBF_nocalib] = xASL_quant_Basil(PWI, x)
+%xASL_quant_Basil Perform quantification using FSL BASIL
+% FORMAT: [CBF_nocalib] = xASL_quant_Basil(PWI, x)
+% 
+% INPUT:
+%   PWI             - image matrix of perfusion-weighted image (REQUIRED)
+%   x               - struct containing pipeline environment parameters (REQUIRED)
+%
+% OUTPUT:
+% CBF_nocalib       - Quantified CBF image
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION: This script performs quantification of the PWI using the FSL Basil pipeline. Final calibration to
+%              physiological units is performed by dividing the quantified PWI by the M0 image/value
```

Comment:
We have to specify what it can do already. For sure single-TI, multi-TI - as defined in the PLD vector. But what else? It cannot for sure call Hadamard or QUASAR yet?! No velocity or acceleration selective?